train.py

```diff
@@ -281,10 +282,14 @@ def train(hyp, opt, device, callbacks):  # hyp is path/to/hyp.yaml or hyp dictio
 
         # Turn off features incompatible with QAT
         if sparse_manager and sparse_manager.qat_active(epoch):
+            sparse_manager.log_console_info(
+                "QAT phase detected. Disabling EMA and AMP if enabled"
+            )
```

Comment:
This will be called during each QAT epoch. I'm currently in favor of keeping it this way rather than tracking QAT on/off for simpler robust coverage of different possible QAT schedules, but can be convinced otherwise

Reply:
why not add `ema.enabled or amp` to the if statement?